operators/kepler-operator/0.4.0/ci.yaml

```diff
@@ -0,0 +1,6 @@
+---
+reviewers:
+  - sustainable-computing-io
+  - husky-parul
```

Comment:
Adding myself (also the author of the PR) is not setting `authorized-changes` label to the PR>

@J0zi @mvalarh

Reply:
This is the expected behaviour when adding a new operator. We will manually add the `authorized-changes` once all tests pass.